src/test/java/edu/hw2/Task3/PopularCommandExecutorTest.java

```diff
@@ -0,0 +1,42 @@
+package edu.hw2.Task3;
+
+import edu.hw2.Task3.connectionmanager.ConnectionManager;
+import edu.hw2.Task3.connectionmanager.DefaultConnectionManager;
+import edu.hw2.Task3.connectionmanager.FaultyConnectionManager;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class PopularCommandExecutorTest {
+
+    @Test
+    @DisplayName("Неисправное соединение через FaultyConnectionManager")
+    void testFaultyConnectionThroughFaultyConnectionManager() {
```

Comment:
`BUG` методы тестов в Джаве называются в snake_case или полу snakeCase хотя я бы его переименовал
```suggestion
    void faultyConnectionManager_return_faultyConnection_test() {
```

Reply:
Поправил